framework/db/pgsql/Schema.php

```diff
@@ -441,11 +441,11 @@ protected function findColumns($table)
                     $column->defaultValue = bindec(trim($column->defaultValue, 'B\''));
                 } elseif (preg_match("/^'(.*?)'::/", $column->defaultValue, $matches)) {
                     $column->defaultValue = $matches[1];
-                } elseif (preg_match('/^(?:\()?(.*?)(?(1)\))(?:::.+)?$/', $column->defaultValue, $matches)) {
-                    if ($matches[1] === 'NULL') {
+                } elseif (preg_match('/^(\()?(.*?)(?(1)\))(?:::.+)?$/', $column->defaultValue, $matches)) {
```

Comment:
The change looks suspicious. Non-capturing grouping was turned into capturing and matches index adjusted accordingly. That should not change anything as far as I can see. Does the test fail without this change?